transport/grpc/peer.go

```diff
@@ -53,7 +53,9 @@ func newPeer(address string, t *Transport) (*grpcPeer, error) {
 			grpc.MaxCallSendMsgSize(t.options.clientMaxSendMsgSize),
 		),
 	}
-	if t.options.clientTLS {
```

Comment:
looks like we should drop this var from the transport options struct

Reply:
I think it's actually fine for them to coexist. clientTLS is connected to a config flag that just says "yes, you can connect to a TLS server." Exposing the whole TLSOptions is more complex and has to be done in code.